pkg/provision/zdb.go

```diff
@@ -358,11 +268,14 @@ func zdbDecommission(ctx context.Context, reservation *Reservation) error {
 		return errors.Wrap(err, "failed to decode reservation schema")
 	}
 
-	containerID, ok := zdbMap.Get(nsID)
-	if !ok {
+	allocation, err := storage.Find(reservation.ID)
```

Comment:
if Find return not found here, does that mean the 0-db namespace is also gone ? Cause do an early return then.

Reply:
It means the namespace directory in 0-db data is gone. I am pretty sure it also means the namespace is gone from zdb itself.

Hence we do an early return once it's 'not found'.